cmake/FindFFTW.cmake

```diff
@@ -0,0 +1,25 @@
+# From: https://github.com/wjakob/layerlab/blob/master/cmake/FindFFTW.cmake
+# FIXME: License concerns?
+
+# - Find FFTW
+# Find the native FFTW includes and library
+#
+#  FFTW_INCLUDES    - where to find fftw3.h
+#  FFTW_LIBRARIES   - List of libraries when using FFTW.
+#  FFTW_FOUND       - True if FFTW found.
+
+if (FFTW_INCLUDES)
+  # Already in cache, be silent
+  set (FFTW_FIND_QUIETLY TRUE)
+endif (FFTW_INCLUDES)
+
+find_path (FFTW_INCLUDES fftw3.h)
```

Comment:
Ok... I think I found the problem.  I would expect this to be `FFTW_INCLUDE_DIR` and `FFTW_LIBRARY_DIRS`.  See `FindSndfile.cmake` as an example.  This is mostly likely why `pkg-config` worked and `find_package` failed.